test/Microsoft.Extensions.Logging.Testing.Tests/project.json

```diff
@@ -14,12 +16,16 @@
     "dnx451": {
```

Comment:
I would prefer to do that in one big pass rather than having some repos with net451 and some with dnx451